packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -904,7 +904,11 @@ class _CupertinoModalPopupRoute<T> extends PopupRoute<T> {
 ///
 /// The `useRootNavigator` argument is used to determine whether to push the
 /// popup to the [Navigator] furthest from or nearest to the given `context`. It
-/// is `false` by default.
+/// is `true` by default.
```

Comment:
It's not the part of the issue itself, but I've noticed that the documentation (default is `false`) doesn't reflect actual state (actual default value is `true`), so I changed it too, but I may be wrong here.